### PR TITLE
Update MM Version on README.md

### PR DESCRIPTION
**Pull request by marianunez**
Updated main Readme to reference the Mattermost version of the plugin.json file to avoid reference to old versions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ This plugin demonstrates the capabilities of a Mattermost plugin. It includes th
 
 Feel free to base your own plugin off this repository, removing or modifying components as needed. If you're already familiar with what plugins can do, consider starting from [mattermost-plugin-starter-template](https://github.com/mattermost/mattermost-plugin-starter-template) instead, which includes the same build framework but omits the demo implementations.
 
-Note that this plugin is authored for Mattermost 5.2 and later, and is not compatible with earlier releases of Mattermost.
+Note that this plugin is authored for the Mattermost version indicated in the `min_server_version` within the [plugin.json](https://github.com/mattermost/mattermost-plugin-demo/blob/2461499b06453b7a37d9ca4aedd4d23d24089047/plugin.json#L6), and is not compatible with earlier releases of Mattermost.
```
Comment (lieut-data) [inline README.md]: nit, I'd remove `the` before `plugin.json`:
```suggestion
Note that this plugin is authored for the Mattermost version indicated in the `min_server_version` within [plugin.json](https://github.com/mattermost/mattermost-plugin-demo/blob/2461499b06453b7a37d9ca4aedd4d23d24089047/plugin.json#L6), and is not compatible with earlier releases of Mattermost.
```

File: README.md [hunk 2]
```diff
@@ -8,6 +8,6 @@ This plugin demonstrates the capabilities of a Mattermost plugin. It includes th
 
 Feel free to base your own plugin off this repository, removing or modifying components as needed. If you're already familiar with what plugins can do, consider starting from [mattermost-plugin-starter-template](https://github.com/mattermost/mattermost-plugin-starter-template) instead, which includes the same build framework but omits the demo implementations.
 
-Note that this plugin is authored for Mattermost 5.2 and later, and is not compatible with earlier releases of Mattermost.
+Note that this plugin is authored for the Mattermost version indicated in the `min_server_version` within [plugin.json](https://github.com/mattermost/mattermost-plugin-demo/blob/2461499b06453b7a37d9ca4aedd4d23d24089047/plugin.json#L6), and is not compatible with earlier releases of Mattermost.
```
Comment (hanzei) [inline README.md]: Why not use https://github.com/mattermost/mattermost-plugin-demo/blob/master/plugin.json instead of a fixed commit?

Comment (lieut-data) [reply README.md]: Good catch, @hanzei! @marianunez, would you be open to updating this accordingly? Also, I was mistaken to suggest this needed but one review: clearly I didn't think through this part of the commit! Thanks for being the second set of eyes @hanzei, even after the fact.

Comment (marianunez) [reply README.md]: Thanks @hanzei! Will update to fix the link

**Pull request merged**